.travis.yml

```diff
@@ -1,10 +1,8 @@
 language: rust
 dist: xenial
 
-rust:
-  - stable
-  - beta
-  - nightly
+rust: []
```

Comment:
Do we even need this anymore since you now specify this in the matrix explicitly? Same for the `env: []` line. I really just don't know travis that well.

Reply:
It is required for `allow_failures` to work properly.

(I don't know who at Travis think this madness is a good design, but it is what it is)

Reply:
BTW,

Documentation: https://docs.travis-ci.com/user/customizing-the-build#rows-that-are-allowed-to-fail

See the `env: # important!` line?

![image](https://user-images.githubusercontent.com/11488886/57085246-ff8a9300-6d25-11e9-97c5-85707e7c06da.png)

Reply:
Ok, thanks for the reference.